### PR TITLE
add auto detect and correct ledger vefification

### PR DESCRIPTION
**Pull request by alexjavabraz**

### CODE REVIEW

File: src/components/ledger/ConfirmLedgerTransaction.vue [hunk 1]
```diff
@@ -220,10 +220,21 @@ export default class ConfirmLedgerTransaction extends Vue {
 
   @Emit('successConfirmation')
   async toTrackId() {
-    const LEDGER_STATUS_CODES = { TRANSACTION_CANCELLED_BY_USER: 27013, DEVICE_LOCKED: 27010 };
+    const LEDGER_STATUS_CODES = {
+      TRANSACTION_CANCELLED_BY_USER: 27013,
+      DEVICE_LOCKED: 27010,
+      USER_EXITED_APP: 27906,
+    };
```
Comment (ronaldsg20) [inline src/components/ledger/ConfirmLedgerTransaction.vue:227]: I think this codes should be on the constants file or an static constant in ´LedgerService´

**Review by ronaldsg20 — CHANGES_REQUESTED** [review]

File: src/components/ledger/ConfirmLedgerTransaction.vue (hunk 1 above)
Comment (alexjavabraz) [reply src/components/ledger/ConfirmLedgerTransaction.vue]: Updated thank you.

**Review by ronaldsg20 — DISMISSED** [review]
LGTM

File: src/components/exchange/ConnectDevice.vue [hunk 2]
```diff
@@ -131,14 +131,17 @@ export default class ConnectDevice extends Vue {
   }
 
   beforeMount() {
-    if (this.walletName === 'Liquality' && !this.showDialog) {
-      this.tryConnectLiquality();
+    if (this.peginTxState.bitcoinWallet === constants.WALLET_LIQUALITY && !this.showDialog) {
+      this.tryConnect();
+    } else if (this.peginTxState.bitcoinWallet === constants.WALLET_LEDGER && !this.showDialog) {
+      this.tryConnect();
```
Comment (josedahlquist) [inline src/components/exchange/ConnectDevice.vue:137]: Seems like you could simplify the if clause
```suggestion
    if ((this.peginTxState.bitcoinWallet === constants.WALLET_LIQUALITY ||
          this.peginTxState.bitcoinWallet === constants.WALLET_LEDGER)
    && !this.showDialog) {
      this.tryConnect();
```

**Review by josedahlquist — APPROVED** [review]

**Pull request closed without merging**